src/pages/reset-password.tsx

```diff
@@ -114,18 +114,14 @@ const ResetPasswordPage = ({ location }) => {
                 <SigninInput
```

Comment:
suggestion: Got some type issues with the token email, wdyt about the following instead?
```ts
let token: { email: string } | null = null
// ... ^ Previously Object type which means that email does not exists on Object
const email = token?.email || parsed?.email as string || ""
```